tests/testthat/test-serializer-csv.R

```diff
@@ -0,0 +1,32 @@
+context("CSV serializer")
+
+test_that("CSV serializes properly", {
+  d <- data.frame(a=1, b=2, c="hi")
```

Comment:
Will need to add a `skip_if_not_installed("readr")` on each of these tests.

Reply:
`?testthat::skip_if_not_installed`
> `skip*` functions are intended for use within `test_that()` blocks.
     All expectations following the `skip*` statement within the same
     `test_that` block will be skipped.

Please call the skip function inside the first line of each `test_that`'s testing expression.

Reply:
sorry, moved now, I have a hand with plaster and it's harder to use the left

Reply:
Thank you for the changes!